include/solvers/eigen_solver.h

```diff
@@ -81,6 +81,16 @@ class EigenSolver : public ReferenceCountedObject<EigenSolver<T> >,
    */
   bool initialized () const { return _is_initialized; }
 
+  /**
+  * \returns \p true if we want to close the matrix before solve,
+  * false otherwise. It is true by default.
+  */
+  bool closeMatrixBeforeSolve() const { return _close_matrix_before_solve; }
```

Comment:
Should be named `close_matrix_before_solve()`. libmesh uses lowercase letters and underscores in function names.